integration/localnet/bootstrap.go

```diff
@@ -1,49 +1,57 @@
 package main
 
 import (
+	"encoding/hex"
 	"encoding/json"
 	"flag"
 	"fmt"
+	"io/ioutil"
 	"os"
 	"path/filepath"
 	"runtime"
 	"time"
 
-	"github.com/plus3it/gorecurcopy"
-	"gopkg.in/yaml.v2"
-
+	"github.com/go-yaml/yaml"
+	"github.com/onflow/flow-go/cmd/bootstrap/cmd"
+	"github.com/onflow/flow-go/cmd/bootstrap/utils"
 	"github.com/onflow/flow-go/cmd/build"
-	"github.com/onflow/flow-go/integration/testnet"
+	"github.com/onflow/flow-go/crypto"
+	testnet "github.com/onflow/flow-go/integration/testnet"
 	"github.com/onflow/flow-go/model/bootstrap"
 	"github.com/onflow/flow-go/model/flow"
+	"github.com/plus3it/gorecurcopy"
 )
 
 const (
-	BootstrapDir               = "./bootstrap"
-	ProfilerDir                = "./profiler"
-	DataDir                    = "./data"
-	TrieDir                    = "./trie"
-	DockerComposeFile          = "./docker-compose.nodes.yml"
-	DockerComposeFileVersion   = "3.7"
-	PrometheusTargetsFile      = "./targets.nodes.json"
-	DefaultCollectionCount     = 3
-	DefaultConsensusCount      = 3
-	DefaultExecutionCount      = 1
-	DefaultVerificationCount   = 1
-	DefaultAccessCount         = 1
-	DefaultUnstakedAccessCount = 0
-	DefaultNClusters           = 1
-	DefaultProfiler            = false
-	DefaultConsensusDelay      = 800 * time.Millisecond
-	DefaultCollectionDelay     = 950 * time.Millisecond
-	AccessAPIPort              = 3569
-	ExecutionAPIPort           = 3600
-	MetricsPort                = 8080
-	RPCPort                    = 9000
-	SecuredRPCPort             = 9001
-	AdminToolPort              = 9002
-	AdminToolLocalPort         = 3700
-	HTTPPort                   = 8000
+	BootstrapDir             = "./bootstrap"
+	ProfilerDir              = "./profiler"
+	DataDir                  = "./data"
+	TrieDir                  = "./trie"
+	DockerComposeFile        = "./docker-compose.nodes.yml"
+	DockerComposeFileVersion = "3.7"
+	PrometheusTargetsFile    = "./targets.nodes.json"
+	DefaultAccessGatewayName = "access_1"
+	DefaultObserverName      = "observer"
+	DefaultMaxObservers      = 1000
+	DefaultCollectionCount   = 3
+	DefaultConsensusCount    = 3
+	DefaultExecutionCount    = 1
+	DefaultVerificationCount = 1
+	DefaultAccessCount       = 1
+	DefaultObserverCount     = 0
+	DefaultNClusters         = 1
+	DefaultProfiler          = false
+	DefaultConsensusDelay    = 800 * time.Millisecond
+	DefaultCollectionDelay   = 950 * time.Millisecond
+	FlowAPIPort              = 3569
```

Comment:
I think you should keep the original name here since there's also an `ExecutionAPIPort`

Reply:
Changed back to AccessAPIPort, thanks for the feedback.